src/test/java/tests/FilterEvalTests.java

```diff
@@ -37,6 +44,17 @@ public boolean matches(String filterExpr, TurboIssue issue) {
         return Qualifier.process(empty, Parser.parse(filterExpr), issue);
     }
 
+    /**
+     * Helper method for testing SemanticException thrown by different qualifiers
+     */
```

Comment:
When exactly is the expected exception is thrown. It's not clear from the method.